src/components/Buttons/Button/Button.tsx

```diff
@@ -7,19 +7,19 @@ import { ButtonType, Context, Size } from '../../../constants';
 // supported although not defined in propTypes
 interface Props extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size'> {
     /** The label of the button */
-    children: string | React.ReactNode;
+    children: string | React.ReactElement;
```

Comment:
Why we can not use here `React.ReactNode`?

Reply:
we had a whole discussion about it on slack, didn't we? `React.Node` can be `null | undefined`, so you practically putting it as optional, even though that's not the intention